web_src/js/components/RepoActionView.vue

```diff
@@ -2,7 +2,13 @@
   <div class="action-view-container">
     <div class="action-view-header">
       <div class="action-info-summary">
-        {{ run.title }}
+        <SvgIcon name="octicon-check-circle-fill" size="20" class="green" v-if="run.status === 'success'"/>
+        <SvgIcon name="octicon-clock" size="20" class="ui text yellow" v-else-if="run.status === 'waiting'"/>
+        <SvgIcon name="octicon-meter" size="20" class="ui text yellow" class-name="job-status-rotate" v-else-if="run.status === 'running'"/>
+        <SvgIcon name="octicon-x-circle-fill" size="20" class="red" v-else/>
```

Comment:
What about separating a `RunStatus` component that takes as parameter the `status` (and perhaps others like `size` and `extraClasses` if we really need these differences)?
Then we automatically keep this one and the one below in sync…

Reply:
It is a good idea. How about merge this PR first, and I will try to do this in another PR.

PS: It seems that `SvgIcon` are not used in:
![image](https://user-images.githubusercontent.com/18380374/222606245-d54a84d8-dc44-4d99-ab42-4a0a11635b5a.png)
![image](https://user-images.githubusercontent.com/18380374/222606256-455c6044-9f0b-4bd8-890e-09e237091843.png)
Are these codes removable?

Reply:
@delvh 
https://github.com/go-gitea/gitea/pull/23259
I created a new PR to do this. I can merge it to this PR, if it looks good.

Reply:
Both OK from me to keep that PR standalone or merge it into this one.